bootstrap/bpdoc/bpdoc_test.go

```diff
@@ -6,6 +6,11 @@ import (
 	"testing"
 )
 
+type simpleProp struct {
```

Comment:
simpleProp sounds like this is one of the structs being tested. What about something like propInfo?

Reply:
Good idea!